### DISCUSSION TITLE
/eco Command Alias

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#5318] [Enhancement]
Add

  # /cmi money $1-
  /money:
    Enabled: true
    Tab: true
  /eco:
    Enabled: false
    Tab: true

By default please

**Comment by mrfloris** [comment]
Hey @Zrips ,

A lot of people realise CMI is a modern solution for their server these days and move away from EssentialsX, and a lot of old 1.8 etc servers also are very used to /eco, and their plugins reflect this. 

It's not too difficult to customize some incidental config files here and there, but certain things (like with this person) who has a LOT of quest config files with eco reward, it might be nice to accommodate everybody by adding `eco` as an alias for pay or money (pref). so it runs fine when a plugin runs things like `- eco give {player} 500` etc. that it's basically the same as money give player 500 through cmi. 

Maybe it's worth considering adding the alias in Alias.yml below /money ?

I recommended that they use aliaseditor to add eco and while this works manually from console or in-game, incorrect permissions can lead to issues, and it seems in this case the quest says: cant find player, while manually running it goes okay.

Having it as alias here would probably resolve that as well. 

I also checked my database and it seems despite everybody knowing we have /pay and /money, that we still have weekly results for players trying out /eco

Thank you for considering.
